### PR TITLE
added lint gha

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains
* Added Lint GHA
* this workflow is REQUIRED for a PR to be mergable

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @nehagup

**Comment by nehagup** [comment]
@Ayush7614 Seems like because of your last PR, the workflow is breaking. Can you please help fix that first? 

I've created a bug for the same here - https://github.com/keploy/keploy/issues/363

**Comment by Ayush7614** [comment]
> @Ayush7614 Seems like because of your last PR, the workflow is breaking. Can you please help fix that first?
> 
> I've created a bug for the same here - keploy/keploy#363

This action works when any change is like ui updation, js code this will works then not in GHA this works

**Review by nehagup — APPROVED** [review]

**Review by nehagup — APPROVED** [review]

**Review by nehagup — APPROVED** [review]

**Comment by nehagup** [comment]
@Ayush7614 the build performance check is still failing.

File: .github/workflows/performance.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - main
+
+jobs:
+  build-time:
+    name: Build Time Performance
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```
Comment (nehagup) [inline .github/workflows/performance.yml:15]: I think it's failing because we need 14.x node version

**Review by nehagup — CHANGES_REQUESTED** [review]

File: .github/workflows/performance.yml (hunk 1 above)
Comment (Ayush7614) [reply .github/workflows/performance.yml:15]: image

Comment (Ayush7614) [reply .github/workflows/performance.yml:15]: we already on node 14 version

**Comment by nehagup** [comment]
@Ayush7614 can you please check why the build performance is failing?

**Comment by nehagup** [comment]
@Ayush7614 are you still working on this?

**Pull request closed without merging**